src/core/seq.js

```
@@ -0,0 +1,124 @@
+  const nums = [];
+  for (let j = 0; j < operands.length; j++) {
+    const v = Number(operands[j]);
+    if (isNaN(v)) {
+      const stderr = `seq: invalid floating point argument: ‘${operands[j]}’\nTry 'seq --help' for more information.\n`;
+      return { stdout: '', stderr, env: Object.assign({}, env, { '?': 1 }) };
+    }
+    nums.push(v);
+  }
+  // assign first, step, last
+  let first, step, last;
+  if (nums.length === 1) {
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)